lib/scope.js

```diff
@@ -1182,37 +1182,98 @@ module.exports = {
   setVariable: async function ( scope, { handle, row }) {
     /* Returns true if the variable was enabled (which
        assumes that it was set correctly if no error happened) */
-    let { disabled, tag, type } = await handle.evaluate( elem => {
+    let { disabled, tag, type } = await handle.evaluate(( elem )=> {
       return {
         disabled: elem.disabled,
         tag: elem.tagName,
         type: elem.type,
       };
     });
 
-    if ( !disabled ) {
-      // da's header covers the top of the section containing the fields which means
-      // sometimes it blocks nodes from clicks. Scroll item to the bottom instead.
-      await handle.evaluate(( elem ) => { elem.scrollIntoView(false); return elem.scrollTop });
-      let set_to = row.value;
-      if ( type === `checkbox` ) { set_to = row.checked; }
-      
-      if ( env_vars.DEBUG && handle ) {
-        let name = await handle.evaluate(( elem ) => {
-          try { elem.nextSibling.style.background = 'Khaki'; }
-          catch(err){ elem.style.background = 'Khaki'; }
-          return elem.getAttribute('for') || elem.getAttribute('name'); 
-        });
-        console.log( `setting elem ${ name }, var ${ row.var } to ${ set_to }` );
-        await scope.page.waitFor( 400 );
+    // If field is disabled, don't set it yet
+    if ( disabled ) { return !disabled; }
+
+    // da's header covers the top of the section containing the fields which means
+    // sometimes it blocks nodes from clicks. Scroll item to the bottom instead.
+    await handle.evaluate(( elem ) => { elem.scrollIntoView(false); return elem.scrollTop });
+    let set_to = row.value;
+    if ( type === `checkbox` ) { set_to = row.checked; }
+
+    if ( env_vars.DEBUG && handle ) {
+      let name = await handle.evaluate(( elem ) => {
+        try { elem.nextSibling.style.background = 'Khaki'; }
+        catch(err){ elem.style.background = 'Khaki'; }
+        return elem.getAttribute('for') || elem.getAttribute('name'); 
+      });
+      await scope.page.waitFor( 400 );
+    }
```

Comment:
I wanted to reduce indentation for this so instead of having everything in an `if`, I made an opposite `if` that would return if needed.